__mocks__/node-persist.js

```diff
@@ -0,0 +1,33 @@
+const storage = jest.genMockFromModule('node-persist');
+
+let data = {};
+
+storage.init = function() {
+  return Promise.resolve();
+};
+
+storage.clear = function() {
+  data = {};
```

Comment:
Isn't this still going to cause problems when test1 clears storage and test2 had written something to storage and was expecting it to be there?

Reply:
Jest runs each test in its own node process (approximately) so each test gets its own mock storage